2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -220,13 +181,13 @@ missing_tallies %>%
 ```
 
 ```{r}
-psm_infiles <- Sys.glob(here('inst/extdata/*_PSMs.txt'))
+psm_infiles <- Sys.glob(here('extdata/*_PSMs.txt'))
 
 names(psm_infiles) <- replicates
 
 ```
 
-```{r, eval=FALSE}
```

Comment:
Previously, this function was taken ages so I suggested running once, saving output and then loading output in subsequent runs. Now it's been sped up 100-fold(!), there's no need to do this so we can remove the `eval=FALSE`